actool/manifest.go

```diff
@@ -250,6 +264,23 @@ func patchManifest(im *schema.ImageManifest) error {
 		}
 	}
 
+	if patchSeccompMode != "" {
```

Comment:
I realized I didn't make it clear and my code is making some assumptions that `set` may be empty. On a second thought, I think I am not sure there are actual usecases for this. Should I make it explicit that `set MUST NOT be empty` and just kill this corner case?